pr_testing/test_multiple_prs.sh

```diff
@@ -1326,7 +1342,9 @@ if [ "X$BUILD_OK" = Xtrue -a "$RUN_TESTS" = "true" ]; then
   fi
   if [ $(echo ${ENABLE_BOT_TESTS} | tr ',' ' ' | tr ' ' '\n' | grep '^GPU$' | wc -l) -gt 0 -a X"${DISABLE_GPU_TESTS}" != X"true" ] ; then
     DO_GPU_TESTS=true
```

Comment:
TODO: figure out how to set DO_GPU_TESTS now that "GPU" is not in EXTRA_RELVAL_TESTS. Maybe keep it and skip when scheduling relvals? L385, L1482,